R/carehomes.R

```diff
@@ -428,14 +439,15 @@ carehomes_initial <- function(info, n_particles, pars) {
   index_N_tot3 <- index[["N_tot3"]][[1L]]
 
   index_S <- index[["S"]]
+  index_S_no_vacc <- index_S[seq_len(length(pars$N_tot))]
```

Comment:
This implies that you have organised your S like:

```
S[i, j]
```

where `i` is the age group and `j` is the vaccination level, right?

Reply:
yes that's right